src/app/devices/deviceContent/components/deviceProperties/deviceProperties.spec.tsx

```diff
@@ -59,22 +61,36 @@ describe('components/devices/deviceProperties', () => {
             ...overrides
         };
 
-        return mountWithLocalization(
-            <DeviceProperties {...props} />, true, [pathname]
+        return (
+            <DeviceProperties {...props} />
         );
     };
 
-    it('matches snapshot while loading', () => {
-        const wrapper = getComponent();
-        expect(wrapper).toMatchSnapshot();
+    it('shows Shimmer while loading', () => {
+        const wrapper = mountWithLocalization(
+            getComponent(), false, true, [pathname]
+        );
         expect(wrapper.find(Shimmer)).toBeDefined();
     });
 
+    it('matches snapshot while interface cannot be found', () => {
+        testSnapshot(getComponent({isLoading: false, twinWithSchema: undefined}));
```

Comment:
When I run the tests locally, I'm seeing not covered branch for when it should be returning `<InterfaceNotFoundMessageBoxContainer/>` so I don't think this is doing overriding the props as you'd expect.